app/models/calendar.rb

```diff
@@ -14,21 +14,34 @@ def self.find(slug)
     end
   end
 
-  attr_reader :slug, :title, :description
+  def self.file_name(slug)
+    slug == "gwyliau-banc" ? "bank-holidays" : slug
```

Comment:
It feels like this should return the file name rather than the slug.

Reply:
Have reengineered to return "scope", which is how the bulk of the code refers to "bank-holidays" vs "when-the-clocks-change" regardless of translation.